android/app/build.gradle

```diff
@@ -83,6 +83,8 @@ project.ext.react = [
 
 apply from: "../../node_modules/react-native/react.gradle"
 
+apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"
```

Comment:
The original project wasn't set up correctly on android, so Icons weren't appearing. That fixes it.